assets/css/generic/_typography.scss

```diff
@@ -0,0 +1,18 @@
+html {
+  color: $base-font-color;
+  font-family: $base-font-family;
+  font-size: $large-font-size;
+  line-height: $base-line-height;
+}
+
+@media (max-height: 960px), (max-width: 1024px) {
+  html {
+    font-size: $base-font-size;
+  }
+}
+
+@media (max-height: 480px), (max-width: 768px) {
+  html {
+    font-size: $small-font-size;
```

Comment:
You could use a CSS custom property (aka variable) to clean this up by not having the redefine `html` and `font-size` for each of these and instead just change the value of the custom property within a query. Nesting the media queries within the `html` block would also clean this up. Something like:

```scss
:root {
  --font-size: #{$large-font-size};
}

html {
  font-size: $large-font-size;

  @media (max-height: 960px), (max-width: 1024px) {
    --font-size: #{$base-font-size};
  }
}
```

Not sure this actually works; it's a bit of pseudo-code.